packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -7,207 +7,282 @@
+                    </DenaliInputGroup>
+                  {{/if}}
+
+                  {{#if (includes this.localDeliveryRule.format.type this.overwriteableFormats)}}
+                    <DenaliInputGroup @label="Overwrite spreadsheet" class="schedule__modal-overwrite">
+                      <DenaliSwitch
+                        class="schedule__modal-overwrite-toggle"
+                        @isEnabled={{this.localDeliveryRule.format.options.overwriteFile}}
+                        @onChange={{this.toggleOverwriteFile}}/>
+                    </DenaliInputGroup>
+                    <p class="help-text m-b-30 is-grey-800">
+                      {{#if this.localDeliveryRule.format.options.overwriteFile}}
+                        Data will be replaced in the same sheet every {{this.localDeliveryRule.frequency}}
+                      {{else}}
+                        A new sheet will be created every {{this.localDeliveryRule.frequency}}
+                      {{/if}}
+                    </p>
+                  {{/if}}
+                  <DenaliInputGroup @label="Status" class="m-b-30 align-items-center">
+                    <DenaliSwitch
+                      class="schedule__modal-disabled-status"
+                      @isEnabled={{not this.localDeliveryRule.isDisabled}}
+                      @onChange={{toggle "isDisabled" this.localDeliveryRule}}/>
+                    <p>
+                      {{#if this.localDeliveryRule.isDisabled}}
+                        Paused
+                      {{else}}
+                        Active
+                      {{/if}}
+                    </p>
+                  </DenaliInputGroup>
+                {{/if}}
+              {{/if}}
+          </div>
+
+          {{#if this.localDeliveryRule}}
+            <div class="schedule__modal-buttons w-full p-24">
+              {{#if (and this.localDeliveryRule.isNew (not (await this.isValidForSchedule)))}}
+                <DenaliButton
+                  class="schedule__modal-cancel-btn m-r-5"
+                  style="ghost"
+                  type="button"
+                  {{on "click" this.closeModal}}
+                >
+                  Close
+                </DenaliButton>
+              {{else if this.errorWhileFetchingRules}}
+                <DenaliButton
+                  class="schedule__modal-cancel-btn m-r-5"
+                  @style="ghost"
+                  disabled={{this.isSaving}}
+                  type="button"
+                  {{on "click" (queue this.closeModal this.revertAll)}}
+                > Cancel
+                </DenaliButton>
+              {{else}}
+                <DenaliButton
+                  class="schedule__modal-save-btn m-r-5"
+                  disabled={{this.disableSave}}
+                  isLoading={{this.isSaving}}
+                  {{on "click" (queue (toggle "isSaving" this) this.doSave)}}
+                >
+                  {{#if this.localDeliveryRule.isNew}}
+                    Save
```

Comment:
If you wind up doing the "clicking on save means that the user is saving all of the schedules" thing, it might make more sense to move the save button to the left panel under the list of schedules